<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RepoSyncDialog</class>
 <widget class="QDialog" name="RepoSyncDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Repo sync</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="forceCB">
     <property name="text">
      <string>[Force] May cause loss of data!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="localOnyCB">
     <property name="text">
      <string>[Local] Only update working tree, don't fetch</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="networkOnlyCB">
     <property name="text">
      <string>[Network] Fetch only, don't update working tree</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="detachCB">
     <property name="text">
      <string>[Detach] Detach projects back to manifest revision</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="pruneCB">
     <property name="text">
      <string>[Prune] Delete refs that no longer exist on the remote</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="verboseBox">
     <property name="text">
      <string>[Verbose] Show all output</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="jobsLabel">
       <property name="text">
        <string>Jobs:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="jobsSpin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item alignment="Qt::AlignBottom">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RepoSyncDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RepoSyncDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
